pyintegration/deephaven2/plot/axistransform.py

```diff
@@ -0,0 +1,51 @@
+#
+#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+""" This module implements the AxisTransform class for performing axis transformation on plots before rendering. """
+
+from typing import List
+
+import jpy
+
+from deephaven2 import DHError
+from deephaven2._wrapper import JObjectWrapper
+
+_JAxisTransform = jpy.get_type("io.deephaven.plot.axistransformations.AxisTransform")
+_JPlottingConvenience = jpy.get_type("io.deephaven.plot.PlottingConvenience")
+
+
+class AxisTransform(JObjectWrapper):
+    """ An axis transformation that is applied before rendering a plot. Axis transforms include logarithms,
+    business time, etc. """
+
+    j_object_type = _JAxisTransform
+
+    @property
+    def j_object(self) -> jpy.JType:
+        return self.j_axis_transform
+
+    def __init__(self, j_axis_transform):
+        self.j_axis_transform = j_axis_transform
+
+
+def axis_transform_names() -> List[str]:
+    """ Returns the names of available axis transforms. """
+    return list(_JPlottingConvenience.axisTransformNames())
+
+
+def axis_transform_by_name(name: str) -> AxisTransform:
```

Comment:
Rename to `axis_transform` to be concise and allow future indexing in different ways?